README.md

```diff
@@ -45,35 +48,35 @@ cd ovsrpro
 git checkout <tag>		# where tag is, for example 17.02.1
 mkdir ovsrpro-build
 cd ovsrpro-build
-cmake ../ovsrpro -DXP_STEP=build
+cmake ../ovsrpro -DXP\_STEP=build
```

Comment:
You're right. I guess there's a bug in the Vim syntax file for Markdown. I'll fix it in a little bit.